test/unittests/test_dd_synthesis_dc.cpp

```diff
@@ -61,7 +61,7 @@ TEST_P(TestDDSynthDc, GenericDDSynthesisDcTest) {
 
     buildTruthTable(*qc, ttqc);
 
-    EXPECT_TRUE(TruthTable::equal(ttqc, tt));
+    EXPECT_TRUE(TruthTable::equal(ttqc, tt) && TruthTable::equal(tt, ttqc));
```

Comment:
Instead of the `&&` just use a separate EXPECT_TRUE. Otherwise there is no way to know which of the checks failed when the expectation fails.